field_encrypt.module

```diff
@@ -85,7 +85,6 @@ function field_encrypt_form_alter(&$form, FormStateInterface $form_state, $form_
       );
 
       // We add functions to process the form when it is saved.
-      $form['actions']['submit']['#submit'][] = 'field_encrypt_update_encryption_form_submit';
```

Comment:
The batch doesn't start automatically on config submit, but rather a message is shown, with a link to run the updates manually. This will then start the batch process.